src/minecraft/kotlin/xyz/wagyourtail/unimined/internal/minecraft/patch/fabric/FabricMinecraftTransformer.kt

```diff
@@ -60,4 +68,84 @@ abstract class FabricMinecraftTransformer(
             "-Dfabric.remapClasspathFile=${intermediaryClasspath}"
         )
     }
+
+    override fun afterRemap(baseMinecraft: MinecraftJar): MinecraftJar {
+        val baseMinecraft = super.afterRemap(baseMinecraft)
+
+        val injections = hashMapOf<String, List<String>>()
+
+        this.collectInterfaceInjections(baseMinecraft, injections)
+
+        return if (injections.isNotEmpty()) {
+            val oldSuffix = if (baseMinecraft.awOrAt != null) baseMinecraft.awOrAt + "+" else ""
+
+            val output = MinecraftJar(
+                baseMinecraft,
+                parentPath = provider.localCache.resolve("fabric").createDirectories(),
+                awOrAt = "${oldSuffix}ii+${injections.hashCode()}"
```

Comment:
I think this should use a short shaw, to match how aw/at's and mapping stubs work

Reply:
Done